site/en/docs/extensions/mv3/service_workers/index.md

```diff
@@ -194,8 +198,8 @@ extension appears and disappears from Chrome's task manager.
 Open the task manager by clicking the Chrome Menu, hovering over more tools and selecting "Task
```

Comment:
```suggestion
Open the task manager by clicking the Chrome menu, hovering over more tools and selecting "Task
```
Suggestion made by @jpmedley in [this comment](https://github.com/GoogleChrome/developer.chrome.com/pull/2107/files#r801775629).